src/app/header/header.component.scss

```diff
@@ -0,0 +1,41 @@
+.app-wrapper {
+    width: 100%;
+    height: 100%;
+    background-color: grey;
```

Comment:
Above was another green backgroud-color. What are these for?

Reply:
It is a GREY background-color ) I just tried to change the background of the page. Sorry, it was just an attempt. I did not remove it. I will not push it next time..